### DISCUSSION TITLE
Env proxy parser error

### DISCUSSION THREAD

**Discussion by aetherwu** [encode/httpx#2650] [Potential Issue]
set http_proxy=http://127.0.0.1:7890 & set https_proxy=https://127.0.0.1:7890
Went wrong around _models.py:

```
if not self.host:
                raise InvalidURL("No host included in URL.")
```

Where as 
set http_proxy=http://127.0.0.1:7890 
Works fine.

I am sorry that I couldn't provide a PR but I hope this issue helps.

**Comment by lovelydinosaur** [comment]
Could you start by providing us with a traceback.
I can't find the line you're referring too.
